packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/styles.ts

```diff
@@ -193,7 +193,13 @@ export function getStylesConfig(wco: WebpackConfigOptions) {
         options: {
           ident: 'embedded',
           plugins: postcssPluginCreator,
-          sourceMap: cssSourceMap && !buildOptions.sourceMap.hidden ? 'inline' : false,
+          sourceMap: cssSourceMap
+            // Never use component css sourcemap when optimizations are on.
+            // It will just increase bundle size without offering good debug experience.
+            && !buildOptions.optimization
```

Comment:
You should actually check it style optimization is enabled. As styles optimization can be disabled.

Reply:
You're right, and that's also what was breaking a server builder test. Thanks!